vcbuild.bat

```diff
@@ -299,7 +302,11 @@ set "msbcpu=/m:2"
 if "%NUMBER_OF_PROCESSORS%"=="1" set "msbcpu=/m:1"
 set "msbplatform=Win32"
 if "%target_arch%"=="x64" set "msbplatform=x64"
-if "%target%"=="Build" if defined no_cctest set target=node
+if "%target%"=="Build" (
+  if defined no_cctest set target=node
+  if "%test_args%"=="" set target=node
+  if defined cctest set target="Build"
+)
```

Comment:
If `cctest` is not built, it's better to delete the executable to make sure it's not possible to run the tests with an outdated `cctest.exe`. Something like: `if "%target%"=="node" if exist "%config%\cctest.exe" del "%config%\cctest.exe"`